plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/extension/MetalavaExtension.kt

```diff
@@ -98,4 +98,15 @@ open class MetalavaExtension {
      * The default is "current".
      */
     var releaseType = "current"
+
+    /**
+     * The directories to search for source files. An exception will be thrown is the named
+     * directories are not direct children of the project root. The default is "src".
+     */
+    var sourcePath = mutableSetOf<String>("src")
```

Comment:
Should this be named as plural? `sourcePaths` for multiple paths?

Reply:
I'm indifferent about whether it's plural or singular. What did drive the choice of the singular usage, here, was that the tool's flag is named  as such, ie:
API sources:

```
--source-path <paths>
                                             One or more directories (separated by `:`) containing source files (within
                                             a package hierarchy). If --strict-input-files, --strict-input-files:warn,
                                             or --strict-input-files:stack are used, files accessed under --source-path
                                             that are not explicitly specified in --source-files are reported as
                                             violations.
```

Typing this now, though, I'll change it to plural.